source/Octopus.Tentacle/Configuration/TentacleConfiguration.cs

```diff
@@ -72,7 +72,7 @@ public string ApplicationDirectory
             get
             {
                 var path = settings.Get<string>(DeploymentApplicationDirectorySettingName);
-                if (string.IsNullOrWhiteSpace(path))
+                if (path is null || string.IsNullOrWhiteSpace(path))
```

Comment:
is this required?

Reply:
@slewis74 maybe we should consider only enabling nullability for netcore?

Reply:
Do you have an example of how to do that? I tried and it went back to possible null reference errors in the compile.

Reply:
I just had a play with the code, and I am confused. It doesn't seem the projects have nullability explicitly enabled, which I thought it means it is NOT?
Also, I noticed `TentacleConfiguration.cs` file has it explicitly ON, why?

Reply:
because that's the pattern that has been used to progressively switch things over. If you switch it on at the project level a ton of things light up. My focus here was to unblock the current issue.